dojo/user/queries.py

```diff
@@ -50,3 +50,7 @@ def get_authorized_users_for_product_and_product_type(users, product, permission
             Q(id__in=product.prod_type.authorized_users.all()) |
             Q(is_superuser=True) |
             Q(is_staff=True))
+
+
+def get_groups_for_user(user):
```

Comment:
This method is not needed anymore, after the suggested change in `user/views.py`